mslib/msui/mss_pyui.py

```diff
@@ -243,7 +243,7 @@ def preload_wms(urls):
                 request = requests.get(base_url)
                 if pdlg.wasCanceled():
                     break
-                wms = wms_control.MSSWebMapService(request.url, version='1.1.1',
+                wms = wms_control.MSSWebMapService(request.url, version='1.3.0',
```

Comment:
We have manually to verify that this version of msui can read a mss server based on MSS-2.0, old version '1.1.1'

We should have a tests for this.

Reply:
I will have to ponder on that. Not sure how this could be incorporated into a test yet.

Reply:
We could have a xml result for mocking the server request and feed this into the gui methods. The mock just replaces the request to the server and has a return parameter with the V1.1.1 XML data.

Reply:
Ultimately I have decided against incorporating this into a test, because that test would not be consistent and will eventually fail. Maybe not next month, maybe not next year, but eventually.
MSS is constantly updating its packages, and at some point we will not be able to run MSS-2.0 anymore because of this. The DLR Server will soon be updated and afterwards the test might fail also.

I have tested both MSS-2.0 and the DLR server by hand, using the new Client.
I tested each layer, made sure everything is drawing correctly, and made sure the communication works without errors.